### PR TITLE
DAT-2156 CuratedContent translations and message fixes

### PR DESCRIPTION
**Pull request by pawelchcki**

### CODE REVIEW

**Comment by pchojnacki** [comment]
Tests are failing because of old base that now contains outdated tests

**Comment by pchojnacki** [comment]
@sqreek CR

File: extensions/wikia/CuratedContent/CuratedContent.i18n.php [hunk 1]
```diff
@@ -61,116 +44,190 @@
...
+	'wikiacuratedcontent-content-description-no-section' => 'crwdns6987:0crwdne6987:0',
+	'wikiacuratedcontent-content-description-items-input' => 'crwdns6988:0{ns:Category}crwdnd6988:0{ns:500}crwdnd6988:0{ns:File}crwdne6988:0',
+	'wikiacuratedcontent-content-articlenotfound-error' => 'crwdns6989:0crwdne6989:0',
+	'wikiacuratedcontent-content-emptylabel-error' => 'crwdns7627:0crwdne7627:0',
+	'wikiacuratedcontent-content-videonotsupported-error' => 'crwdns6991:0crwdne6991:0',
+	'wikiacuratedcontent-content-notsupportedtype-error' => 'crwdns6992:0crwdne6992:0',
+	'wikiacuratedcontent-content-nocategoryintag-error' => 'crwdns6993:0crwdne6993:0',
+	'wikiacuratedcontent-featured-section-name' => 'crwdns6994:0crwdne6994:0',
+);
+
```
Comment (sqreek) [inline extensions/wikia/CuratedContent/CuratedContent.i18n.php]: this section looks strange

**Comment by sqreek** [comment]
except Acholi dialect LGTM

**Pull request merged**